ambry-rest/src/main/java/com.github.ambry.rest/NettyFrontendServerFactory.java

```diff
@@ -35,16 +35,16 @@
  * Sets up all the supporting cast required for the operation of {@link NettyServer} and returns a new instance on
  * {@link #getNioServer()}.
  */
-public class NettyServerFactory implements NioServerFactory {
-  private static final Logger LOGGER = LoggerFactory.getLogger(NettyServerFactory.class);
+public class NettyFrontendServerFactory implements NioServerFactory {
```

Comment:
I would rename this to "FrontendNettyServerFactory".  FrontendServer looks weird.

Reply:
ack.

Reply:
May be FrontendNettyFactory, StorageNettyFactory.